packages/account/tests/account.e2e.spec.ts

```diff
@@ -90,20 +86,73 @@ describe("Account Tests", () => {
     };
 
     const balance = (await checkBalance(publicClient, recipient, nftAddress)) as bigint;
-    const partialUserOp = await smartWallet.buildUserOp([transaction]);
 
-    const paymasterData = await paymaster.getPaymasterAndData(partialUserOp, {
-      mode: PaymasterMode.SPONSORED,
+    const maticBalanceBefore = publicClient.getBalance({ address: await smartWallet.getAddress() });
```

Comment:
There's a utils function for this in the tests directory

Reply:
checkBalance used above

Reply:
const balance = (await checkBalance(publicClient, await smartWallet.getAddress() )) as bigint;

Reply:
Should work for nativeToken, though I've not tested it